Reports/report-docx-criticalAndHighIncidentsReport.json

```diff
@@ -0,0 +1,374 @@
+{
+  "id": "docxCriticalAndHighIncidentsReport",
+  "name": "Critical and High incidents",
+  "description": "All critical and high severity incidents that may need the analyst attention.",
```

Comment:
Isn't this redundant let alone confusing ? This details is the details of the report. The symbol near it shows clearly what format this is going to be. I think we should remove it from the description at all..